services/wallet/reader.go

```diff
@@ -195,6 +198,19 @@ func (r *Reader) startWalletEventsWatcher() {
 	r.walletEventsWatcher = walletevent.NewWatcher(r.walletFeed, walletEventCb)
 
 	r.walletEventsWatcher.Start()
+
+        r.walletFeed.Subscribe(SIGNAL_MANUAL_RELOAD_TRIGGERED, handleManualReloadTriggered)
+}
+
+func handleManualReloadTriggered(event walletevent.Event) {
```

Comment:
There is no receiver set for this function, but in the body, you're calling `r.`, that won't compile.